frontend/tests/jest/tests/ProductSection.test.tsx

```diff
@@ -118,6 +118,33 @@ describe('ProductSection Tests', () => {
          (expect(note) as any).not.toBeInTheDocument();
          (expect(disclaimer) as any).not.toBeInTheDocument();
       });
+
+      test('compatibility renders', async () => {
+         renderWithAppContext(
+            <ProductSection
+               product={getMockProduct()}
+               selectedVariant={getMockProductVariant()}
+               onVariantChange={jest.fn()}
+               internationalBuyBox={null}
+            />
+         );
+
+         expect(
+            screen.getByTestId('product-compatibility-dropdown')
+         ).not.toBeVisible();
+
+         const compatibilityButton = await screen.findByTestId(
+            'product-compatibility-dropdown-button'
+         );
+
+         await act(() => compatibilityButton.click());
+
+         waitFor(() =>
```

Comment:
Do we need this `waitFor` if we use `findBy`? `findBy` is a combination of `waitFor` and `getBy`.

Reply:
When I chained the `click()` call to the find call above, I could drop the `waitFor`. 👍